index.d.ts

```diff
@@ -487,7 +490,19 @@ declare namespace Eris {
     color?: number;
     hoist?: boolean;
     mentionable?: boolean;
+    managed?: boolean;
```

Comment:
I don't believe this is a valid option to pass.

Reply:
Not a valid option? It might not be able to be changed, but it is definitely sent
![image](https://user-images.githubusercontent.com/17226394/70070346-12658e80-15b9-11ea-9a07-a97e36109d6f.png)

Reply:
Oh, I see what you mean, I'm not 100% sure, let me go test things.

Reply:
What you are editing is RoleOptions which is used for creating roles. Bots can't create managed roles so passing a managed thing is not a thing.

Reply:
Doesn't make a different, so basically isn't a valid option
![image](https://user-images.githubusercontent.com/17226394/70070988-2b227400-15ba-11ea-8b51-fa52c68f8d69.png)